apps/site-app/src/app.tsx

```diff
@@ -67,10 +64,3 @@ if (module.hot) {
 }
 
 render(translations);
-
-// Install ServiceWorker and AppCache in the end since
-// it's not most important operation and if main code fails,
-// we do not want it installed
-if (process.env.NODE_ENV === 'production') {
-  require('offline-plugin/runtime').install();
-}
```

Comment:
offline-plugin is not compatible with Webpack5, but at this point we aren't using it anyway